app/views/home/index.html.erb

```diff
@@ -1,128 +0,0 @@
-<div class="d-flex flex-column">
```

Comment:
Why removed?

Reply:
because it is shifted to companies index page by Usman.